### PR TITLE
Remove snyk-compliance package.json files, revert edits to lerna.json

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

This PR is a maintenance fix to remove snyk compliance packages post v16.     

Addresses (https://bugs.earthdata.nasa.gov/browse/CUMULUS-2946)

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -5,9 +5,6 @@
     "example",
     "example/lambdas/*",
     "example/scripts/*",
-    "lambdas/db-migration",
-    "lambdas/db-provision-user-database",
-    "lambdas/sqs-message-remover",
```
Comment (npauzenga) [inline lerna.json:10]: What does this lerna.json do? We still have these lambdas referenced here. I think I just don't understand what this is doing.

**Review by npauzenga — COMMENTED** [review]
Just a quick question about the lerna.json and how that relates.

File: lerna.json (hunk 1 above)
Comment (Jkovarik) [reply lerna.json:10]: So these lines are part of the 'packages' configuration object (array) that tell lerna what the list of monorepo packages are that should be included when lifecycle scripts are run.  

In this case, we're removing the specific package directories and setting it to look for lambda/*

Comment (npauzenga) [reply lerna.json:10]: Cool, totally makes sense

**Review by npauzenga — APPROVED** [review]
Thanks @Jkovarik this looks great (once the Orca dependency is resolved)

**Pull request merged**